ament_cmake_version/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(ament_cmake_version NONE)
+
+find_package(ament_cmake_core REQUIRED)
+
+ament_package(CONFIG_EXTRAS "ament_cmake_version-extras.cmake")
+
+if(BUILD_TESTING)
+    find_package(ament_cmake_lint_cmake REQUIRED)
+    ament_lint_cmake(${CMAKE_CURRENT_SOURCE_DIR})
+endif()
```

Comment:
This package can't use the `ament_cmake_lint_cmake` package since that would create a cyclic dependency. (It also doesn't declare that dependency in its manifest which would probable expose the cycle.)